tests/virt_autotest/login_console.pm

```diff
@@ -35,7 +35,12 @@ sub login_to_console {
     if (match_has_tag('prague-pxe-menu')) {
         send_key 'ret';
 
-        assert_screen([qw(grub2 grub1)], 30);
+        assert_screen([qw(grub2 grub1)], 60);
+    }
+    else {
+        # Wait for bootload for the first time.
+        $self->wait_grub(bootloader_time => 210);
+        send_key 'ret';
```

Comment:
@pdostal Would you please explain why this else branch needed? Actually I just removed it to let xen menuentry selection work last week with https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/6948/files. Now openqa xen tests fail again, see https://openqa.suse.de/tests/2534928#step/login_console/17.

Reply:
Fix in https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/7009. Please raise comments if any concern.